src/components/Carousel/index.js

```diff
@@ -167,6 +170,26 @@ export default class Carousel extends PureComponent {
       'mc-carousel__slider': true,
     })
 
+    let peekStyles
+    if (peek) {
+      peekStyles = {
```

Comment:
I think you'll only need to set the `background` styles here if everything else is already coming from the modifiers.scss file. We do this on the `TileOverlay` component: https://github.com/yankaindustries/mc-components/blob/master/src/components/TileOverlay/index.js

Reply:
Disregard - I remember the convo in slack now about not being able to style `:after` pseudo elements.